tests/broker_version_tests.py

```diff
@@ -46,11 +48,18 @@ def test_it(version, deploy=True, conf={}, rdkconf={}, tests=None,
     cluster.start(timeout=30)
 
     if conf.get('test_mode', '') == 'bash':
-        cmd = 'bash --rcfile <(cat ~/.bashrc; echo \'PS1="[TRIVUP:%s@%s] \\u@\\h:\\w$ "\')' % (  # noqa: E501
-            cluster.name, version)
+        if inherit_env:
+            env = dict(os.environ, **rdkafka.env)
+        else:
+            env = dict(rdkafka.env)
+        trivup = f'[TRIVUP:{cluster.name}@{version}] '
+        PS1 = ((trivup + env['PS1']) if 'PS1' in env
+               else trivup + '\\u@\\h:\\w$ ')\
+            .translate(str.maketrans({'\'': '\\\''}))
```

Comment:
What does this translate do and why do we need it?

Reply:
It quotes any single quotes that you have in your existing `env['PS1']`

Reply:
Was it a bug earlier?

Reply:
no, before it was replacing the PS1, not adding a prefix

Reply:
I saw that this is not replacing 2 backslashs. Is this expected?

Reply:
Here it's only replacing single quotes because later it's written inside single quotes. In:
```c
cmd = f'bash --rcfile <(cat ~/.bashrc; echo \'PS1="{PS1}"\')'
```